src/Accord.Bot/CommandGroups/UserChannelBlockingCommandGroup.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Accord.Bot.Helpers;
+using Accord.Services.Helpers;
+using Accord.Services.UserChannelBlocks;
+using MediatR;
+using Remora.Commands.Attributes;
+using Remora.Commands.Groups;
+using Remora.Discord.API.Abstractions.Objects;
+using Remora.Discord.API.Abstractions.Rest;
+using Remora.Discord.API.Objects;
+using Remora.Discord.Commands.Conditions;
+using Remora.Discord.Commands.Contexts;
+using Remora.Discord.Commands.Parsers;
+using Remora.Results;
+
+namespace Accord.Bot.CommandGroups
+{
+    [Group("channel")]
+    public class UserChannelBlockingCommandGroup : CommandGroup
```

Comment:
Could we make the domain consistent, i.e. you've called it hide in some places, block in others. I like hiding better personally. So this might become `UserChannelHidingCommandGroup`

Reply:
resolved